### ISSUE TITLE
Ask for help

### ISSUE THREAD

**Issue by A-Little-Nut** [appium/appium#13984]
## The problem

selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80

My chrome version is  80.0.3987.116
My webview chrome version is 66.0.3359.126
which version of chromedriver should I use in appium?

## Appium logs

The server is running
[Appium] Welcome to Appium v1.15.1
[Appium] Non-default server args:
[Appium]   address: 127.0.0.1
[Appium]   allowInsecure: {
[Appium]   }
[Appium]   denyInsecure: {
[Appium]   }
[Appium] Appium REST http interface listener started on 127.0.0.1:4723
[HTTP] --> POST /wd/hub/session
[HTTP] {"capabilities":{"firstMatch":[{"platformName":"Android","appium:platformVersion":"10.0","appium:deviceName":"S2D0218B09012250","appium:appPackage":"com.tencent.mm","appium:appActivity":".ui.LauncherUI","appium:automationName":"Uiautomator2","appium:unicodeKeyboard":true,"appium:resetKeyboard":true,"appium:noReset":true,"appium:chromeOptions":{"androidProcess":"com.tencent.mm:tools"}}]},"desiredCapabilities":{"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"}}}
[W3C] Calling AppiumDriver.createSession() with args: [{"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"}},null,{"firstMatch":[{"platformName":"Android","appium:platformVersion":"10.0","appium:deviceName":"S2D0218B09012250","appium:appPackage":"com.tencent.mm","appium:appActivity":".ui.LauncherUI","appium:automationName":"Uiautomator2","appium:unicodeKeyboard":true,"appium:resetKeyboard":true,"appium:noReset":true,"appium:chromeOptions":{"androidProcess":"com.tencent.mm:tools"}}]}]
[BaseDriver] Event 'newSessionRequested' logged at 1582561650999 (00:27:30 GMT+0800 (中国标准时间))
[Appium] Appium v1.15.1 creating new AndroidUiautomator2Driver (v1.37.2) session
[BaseDriver] W3C capabilities and MJSONWP desired capabilities were provided
[BaseDriver] Creating session with W3C capabilities: {
[BaseDriver]   "alwaysMatch": {
[BaseDriver]     "platformName": "Android",
[BaseDriver]     "appium:platformVersion": "10.0",
[BaseDriver]     "appium:deviceName": "S2D0218B09012250",
[BaseDriver]     "appium:appPackage": "com.tencent.mm",
[BaseDriver]     "appium:appActivity": ".ui.LauncherUI",
[BaseDriver]     "appium:automationName": "Uiautomator2",
[BaseDriver]     "appium:unicodeKeyboard": true,
[BaseDriver]     "appium:resetKeyboard": true,
[BaseDriver]     "appium:noReset": true,
[BaseDriver]     "appium:chromeOptions": {
[BaseDriver]       "androidProcess": "com.tencent.mm:tools"
[BaseDriver]     }
[BaseDriver]   },
[BaseDriver]   "firstMatch": [
[BaseDriver]     {}
[BaseDriver]   ]
[BaseDriver] }
[BaseDriver] Session created with session id: 1f56446c-f400-402a-8ae0-278c1ce22765
[UiAutomator2] Starting 'com.tencent.mm' directly on the device
[ADB] Using 'adb.exe' from 'D:\APP\Android-sdk\platform-tools\adb.exe'
[AndroidDriver] Retrieving device list
[ADB] Trying to find a connected android device
[ADB] Getting connected devices...
[ADB] Connected devices: [{"udid":"S2D0218B09012250","state":"device"}]
[AndroidDriver] Looking for a device with Android '10.0'
[ADB] Setting device id to S2D0218B09012250
[ADB] Getting device platform version
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell getprop ro.build.version.release'
[ADB] Current device property 'ro.build.version.release': 10
[AndroidDriver] Using device: S2D0218B09012250
[ADB] Using 'adb.exe' from 'D:\APP\Android-sdk\platform-tools\adb.exe'
[ADB] Setting device id to S2D0218B09012250
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell getprop ro.build.version.sdk'
[ADB] Current device property 'ro.build.version.sdk': 29
[ADB] Device API level: 29
[UiAutomator2] Relaxing hidden api policy
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell settings put global hidden_api_policy_pre_p_apps 1'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell settings put global hidden_api_policy_p_apps 1'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell settings put global hidden_api_policy 1'
[AndroidDriver] No app sent in, not parsing package/activity
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 wait-for-device'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell echo ping'
[AndroidDriver] Pushing settings apk to device...
[ADB] Getting install status for io.appium.settings
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys package io.appium.settings'
[ADB] 'io.appium.settings' is installed
[ADB] Getting package info for 'io.appium.settings'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys package io.appium.settings'
[ADB] Cannot extract apk info using apkanalyzer. Falling back to aapt. Original error: Could not find 'apkanalyzer.bat' in ["D:\\APP\\Android-sdk\\platform-tools\\apkanalyzer.bat","D:\\APP\\Android-sdk\\emulator\\apkanalyzer.bat","D:\\APP\\Android-sdk\\tools\\apkanalyzer.bat","D:\\APP\\Android-sdk\\tools\\bin\\apkanalyzer.bat","D:\\APP\\Android-sdk\\build-tools\\29.0.3\\apkanalyzer.bat"]. Do you have Android Build Tools installed at 'D:\APP\Android-sdk'?
[ADB] Using 'aapt.exe' from 'D:\APP\Android-sdk\build-tools\29.0.3\aapt.exe'
[ADB] Error "W/ziparchive( 5904): Unable to open 'C:\Users\����\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk': Illegal byte sequence
[ADB] W/zipro   ( 5904): Error opening archive C:\Users\����\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk: I/O error
[ADB] ERROR: dump failed because no AndroidManifest.xml found
[ADB] " while getting badging info
[ADB] Cannot read version codes of 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk' and/or 'io.appium.settings'
[ADB] Cannot read version names of 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk' and/or 'io.appium.settings'
[ADB] The current install state of 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk' is unknown. Installing anyway
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell 'ls -t -1 /data/local/tmp/appium_cache 2>&1 || echo _ERROR_''
[ADB] The count of applications in the cache: 3
[ADB] The application at 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\io.appium.settings\apks\settings_apk-debug.apk' is already cached to '/data/local/tmp/appium_cache/3d97225b036c28750562ef8962defee6b1af19a4.apk'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell pm install -r -g /data/local/tmp/appium_cache/3d97225b036c28750562ef8962defee6b1af19a4.apk'
[ADB] The installation of 'settings_apk-debug.apk' took 4.428s
[ADB] Install command stdout: Success
[ADB] Getting IDs of all 'io.appium.settings' processes
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell 'pgrep --help; echo $?''
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell pgrep -f io\\.appium\\.settings'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell am start -n io.appium.settings/.Settings -a android.intent.action.MAIN -c android.intent.category.LAUNCHER'
[ADB] Getting IDs of all 'io.appium.settings' processes
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell pgrep -f io\\.appium\\.settings'
[ADB] Getting IDs of all 'io.appium.settings' processes
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell pgrep -f io\\.appium\\.settings'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell appops set io.appium.settings android\:mock_location allow'
[Logcat] Starting logcat capture
[AndroidDriver] Enabling Unicode keyboard support
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell settings get secure default_input_method'
[AndroidDriver] Unsetting previous IME io.appium.settings/.UnicodeIME
[AndroidDriver] Setting IME to 'io.appium.settings/.UnicodeIME'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell ime enable io.appium.settings/.UnicodeIME'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell ime set io.appium.settings/.UnicodeIME'
[ADB] Getting install status for io.appium.uiautomator2.server
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys package io.appium.uiautomator2.server'
[ADB] 'io.appium.uiautomator2.server' is installed
[ADB] Getting package info for 'io.appium.uiautomator2.server'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys package io.appium.uiautomator2.server'
[ADB] Cannot extract apk info using apkanalyzer. Falling back to aapt. Original error: Could not find 'apkanalyzer.bat' in ["D:\\APP\\Android-sdk\\platform-tools\\apkanalyzer.bat","D:\\APP\\Android-sdk\\emulator\\apkanalyzer.bat","D:\\APP\\Android-sdk\\tools\\apkanalyzer.bat","D:\\APP\\Android-sdk\\tools\\bin\\apkanalyzer.bat","D:\\APP\\Android-sdk\\build-tools\\29.0.3\\apkanalyzer.bat"]. Do you have Android Build Tools installed at 'D:\APP\Android-sdk'?
[ADB] Error "W/ziparchive( 8368): Unable to open 'C:\Users\����\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk': Illegal byte sequence
[ADB] W/zipro   ( 8368): Error opening archive C:\Users\����\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk: I/O error
[ADB] ERROR: dump failed because no AndroidManifest.xml found
[ADB] " while getting badging info
[ADB] Cannot read version codes of 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk' and/or 'io.appium.uiautomator2.server'
[ADB] Cannot read version names of 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk' and/or 'io.appium.uiautomator2.server'
[UiAutomator2] io.appium.uiautomator2.server installation state: unknown
[ADB] Checking app cert for C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk
[ADB] Using 'apksigner.bat' from 'D:\APP\Android-sdk\build-tools\29.0.3\apksigner.bat'
[ADB] Starting 'D:\APP\Android-sdk\build-tools\29.0.3\apksigner.bat' with args '["verify","--print-certs","C:\\Users\\蒋磊\\AppData\\Local\\Programs\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-uiautomator2-server\\apks\\appium-uiautomator2-server-v4.3.0.apk"]'
[ADB] apksigner stdout: Signer #1 certificate DN: EMAILADDRESS=android@android.com, CN=Android, OU=Android, O=Android, L=Mountain View, ST=California, C=US
[ADB] Signer #1 certificate SHA-256 digest: a40da80a59d170caa950cf15c18c454d47a39b26989d8b640ecd745ba71bf5dc
[ADB] Signer #1 certificate SHA-1 digest: 61ed377e85d386a8dfee6b864bd85b0bfaa5af81
[ADB] Signer #1 certificate MD5 digest: e89b158e4bcf988ebd09eb83f5378e87
[ADB] 
[ADB] 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk' is already signed.
[ADB] Getting install status for io.appium.uiautomator2.server.test
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys package io.appium.uiautomator2.server.test'
[ADB] 'io.appium.uiautomator2.server.test' is installed
[ADB] Checking app cert for C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk
[ADB] Starting 'D:\APP\Android-sdk\build-tools\29.0.3\apksigner.bat' with args '["verify","--print-certs","C:\\Users\\蒋磊\\AppData\\Local\\Programs\\Appium\\resources\\app\\node_modules\\appium\\node_modules\\appium-uiautomator2-server\\apks\\appium-uiautomator2-server-debug-androidTest.apk"]'
[ADB] apksigner stdout: Signer #1 certificate DN: EMAILADDRESS=android@android.com, CN=Android, OU=Android, O=Android, L=Mountain View, ST=California, C=US
[ADB] Signer #1 certificate SHA-256 digest: a40da80a59d170caa950cf15c18c454d47a39b26989d8b640ecd745ba71bf5dc
[ADB] Signer #1 certificate SHA-1 digest: 61ed377e85d386a8dfee6b864bd85b0bfaa5af81
[ADB] Signer #1 certificate MD5 digest: e89b158e4bcf988ebd09eb83f5378e87
[ADB] 
[ADB] 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk' is already signed.
[UiAutomator2] Server packages are not going to be (re)installed
[UiAutomator2] Waiting up to 30000ms for services to be available
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell pm list instrumentation'
[UiAutomator2] Instrumentation target 'io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner' is available
[UiAutomator2] Forwarding UiAutomator2 Server port 6790 to 8223
[ADB] Forwarding system: 8223 to device: 6790
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 forward tcp\:8223 tcp\:6790'
[UiAutomator2] No app capability. Assuming it is already on the device
[UiAutomator2] Performing shallow cleanup of automation leftovers
[UiAutomator2] The following obsolete sessions are still running: ["c6569dc2-0bec-41ad-9052-184704cf66a7"]
[UiAutomator2] Cleaning up the obsolete sessions
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell am force-stop io.appium.uiautomator2.server.test'
[Instrumentation] .
[UiAutomator2] Starting UIAutomator2 server 4.3.0
[UiAutomator2] Using UIAutomator2 server from 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.3.0.apk' and test from 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[UiAutomator2] Waiting up to 30000ms for UiAutomator2 to be online...
[ADB] Creating ADB subprocess with args: ["-P",5037,"-s","S2D0218B09012250","shell","am","instrument","-w","io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner"]
[Instrumentation] INSTRUMENTATION_RESULT: shortMsg=Process crashed.
[Instrumentation] INSTRUMENTATION_CODE: 0
[Instrumentation] The process has exited with code 0
[Instrumentation] io.appium.uiautomator2.server.test.AppiumUiAutomator2Server:
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://localhost:8223/wd/hub/status] with no body
[WD Proxy] Got an unexpected response with status undefined: {"code":"ECONNRESET"}
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://localhost:8223/wd/hub/status] with no body
[WD Proxy] Got response with status 200: {"sessionId":"None","value":{"ready":true,"message":"UiAutomator2 Server is ready to accept commands"}}
[UiAutomator2] The initialization of the instrumentation process took 2123ms
[WD Proxy] Matched '/session' to command name 'createSession'
[WD Proxy] Proxying [POST /session] to [POST http://localhost:8223/wd/hub/session] with body: {"capabilities":{"firstMatch":[{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"}},"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"},"deviceUDID":"S2D0218B09012250"}],"alwaysMatch":{}}}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","capabilities":{"firstMatch":[{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"}},"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"},"deviceUDID":"S2D0218B09012250"}],"alwaysMatch":{}}}}
[WD Proxy] Determined the downstream protocol as 'W3C'
[WD Proxy] Proxying [GET /appium/device/info] to [GET http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/appium/device/info] with no body
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"androidId":"84c7e9e4fa3201c4","manufacturer":"HUAWEI","model":"LYA-AL00","brand":"HUAWEI","apiVersion":"29","platformVersion":"10","carrierName":"中国移动","realDisplaySize":"1080x2340","displayDensity":480,"networks":[{"type":0,"typeName":"MOBILE","subtype":13,"subtypeName":"LTE","isConnected":true,"detailedState":"CONNECTED","state":"CONNECTED","extraInfo":"cmnet","isAvailable":true,"isFailover":false,"isRoaming":false,"capabilities":{"transportTypes":"NET_CAPABILITY_MMS","networkCapabilities":"","linkUpstreamBandwidthKbps":51200,"linkDownBandwidthKbps":102400,"signalStrength":-2147483648,"networkSpecifier":"1","SSID":null}}],"locale":"zh_CN_#Hans","timeZone":"Asia\/Shanghai"}}
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys window'
[AndroidDriver] Screen already unlocked, doing nothing
[UiAutomator2] Starting 'com.tencent.mm/.ui.LauncherUI and waiting for 'com.tencent.mm/.ui.LauncherUI'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell am start -W -n com.tencent.mm/.ui.LauncherUI -S'
[WD Proxy] Proxying [GET /appium/device/pixel_ratio] to [GET http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/appium/device/pixel_ratio] with body: {}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":3}
[WD Proxy] Matched '/appium/device/system_bars' to command name 'getSystemBars'
[WD Proxy] Proxying [GET /appium/device/system_bars] to [GET http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/appium/device/system_bars] with body: {}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"statusBar":75}}
[WD Proxy] Matched '/window/current/size' to command name 'getWindowSize'
[WD Proxy] Proxying [GET /window/current/size] to [GET http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/window/current/size] with body: {}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"height":2265,"width":1080}}
[Appium] New AndroidUiautomator2Driver session created successfully, session 1f56446c-f400-402a-8ae0-278c1ce22765 added to master session list
[BaseDriver] Event 'newSessionStarted' logged at 1582561666617 (00:27:46 GMT+0800 (中国标准时间))
[W3C (1f56446c)] Cached the protocol value 'W3C' for the new session 1f56446c-f400-402a-8ae0-278c1ce22765
[W3C (1f56446c)] Responding to client with driver.createSession() result: {"capabilities":{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"platformName":"Android","platformVersion":"10.0","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"}},"platformName":"Android","platformVersion":"10","deviceName":"S2D0218B09012250","appPackage":"com.tencent.mm","appActivity":".ui.LauncherUI","automationName":"Uiautomator2","unicodeKeyboard":true,"resetKeyboard":true,"noReset":true,"chromeOptions":{"androidProcess":"com.tencent.mm:tools"},"deviceUDID":"S2D0218B09012250","deviceApiLevel":29,"deviceScreenSize":"1080x2340","deviceScreenDensity":480,"deviceModel":"LYA-AL00","deviceManufacturer":"HUAWEI","pixelRatio":3,"statBarHeight":75,"viewportRect":{"l...
[HTTP] <-- POST /wd/hub/session 200 15620 ms - 1127
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/timeouts
[HTTP] {"implicit":10000}
[W3C (1f56446c)] Calling AppiumDriver.timeouts() with args: [null,null,null,null,10000,"1f56446c-f400-402a-8ae0-278c1ce22765"]
[BaseDriver] W3C timeout argument: {"implicit":10000}}
[BaseDriver] Set implicit wait to 10000ms
[W3C (1f56446c)] Responding to client with driver.timeouts() result: null
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/timeouts 200 3 ms - 14
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element
[HTTP] {"using":"id","value":"com.tencent.mm:id/r_"}
[W3C (1f56446c)] Calling AppiumDriver.findElement() with args: ["id","com.tencent.mm:id/r_","1f56446c-f400-402a-8ae0-278c1ce22765"]
[BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[BaseDriver] Waiting up to 10000 ms for condition
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/r_","context":"","multiple":false}
[WD Proxy] Got an unexpected response with status 404: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters","stacktrace":"io.appium.uiautomator2.common.exceptions.ElementNotFoundException: An element could not be located on the page using the given search parameters\n\tat io.appium.uiautomator2.handler.FindElement.safeHandle(FindElement.java:78)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:38)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:252)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:242)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:44)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.chann...
[W3C] Matched W3C error code 'no such element' to NoSuchElementError
[BaseDriver] Waited for 533 ms so far
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/r_","context":"","multiple":false}
[WD Proxy] Got an unexpected response with status 404: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters","stacktrace":"io.appium.uiautomator2.common.exceptions.ElementNotFoundException: An element could not be located on the page using the given search parameters\n\tat io.appium.uiautomator2.handler.FindElement.safeHandle(FindElement.java:78)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:38)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:252)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:242)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:44)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.chann...
[W3C] Matched W3C error code 'no such element' to NoSuchElementError
[BaseDriver] Waited for 1071 ms so far
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/r_","context":"","multiple":false}
[WD Proxy] Got an unexpected response with status 404: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters","stacktrace":"io.appium.uiautomator2.common.exceptions.ElementNotFoundException: An element could not be located on the page using the given search parameters\n\tat io.appium.uiautomator2.handler.FindElement.safeHandle(FindElement.java:78)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:38)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:252)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:242)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:44)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.chann...
[W3C] Matched W3C error code 'no such element' to NoSuchElementError
[BaseDriver] Waited for 2160 ms so far
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/r_","context":"","multiple":false}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"ELEMENT":"e802fe1f-d92c-4457-a2fd-a0e749d43d16","element-6066-11e4-a52e-4f735466cecf":"e802fe1f-d92c-4457-a2fd-a0e749d43d16"}}
[W3C (1f56446c)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"e802fe1f-d92c-4457-a2fd-a0e749d43d16","ELEMENT":"e802fe1f-d92c-4457-a2fd-a0e749d43d16"}
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element 200 3096 ms - 137
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/e802fe1f-d92c-4457-a2fd-a0e749d43d16/click
[HTTP] {"id":"e802fe1f-d92c-4457-a2fd-a0e749d43d16"}
[W3C (1f56446c)] Calling AppiumDriver.click() with args: ["e802fe1f-d92c-4457-a2fd-a0e749d43d16","1f56446c-f400-402a-8ae0-278c1ce22765"]
[WD Proxy] Matched '/element/e802fe1f-d92c-4457-a2fd-a0e749d43d16/click' to command name 'click'
[WD Proxy] Proxying [POST /element/e802fe1f-d92c-4457-a2fd-a0e749d43d16/click] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element/e802fe1f-d92c-4457-a2fd-a0e749d43d16/click] with body: {"element":"e802fe1f-d92c-4457-a2fd-a0e749d43d16"}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":null}
[W3C (1f56446c)] Responding to client with driver.click() result: null
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/e802fe1f-d92c-4457-a2fd-a0e749d43d16/click 200 545 ms - 14
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element
[HTTP] {"using":"id","value":"com.tencent.mm:id/m7"}
[W3C (1f56446c)] Calling AppiumDriver.findElement() with args: ["id","com.tencent.mm:id/m7","1f56446c-f400-402a-8ae0-278c1ce22765"]
[BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[BaseDriver] Waiting up to 10000 ms for condition
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/m7","context":"","multiple":false}
[WD Proxy] Got an unexpected response with status 404: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters","stacktrace":"io.appium.uiautomator2.common.exceptions.ElementNotFoundException: An element could not be located on the page using the given search parameters\n\tat io.appium.uiautomator2.handler.FindElement.safeHandle(FindElement.java:78)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:38)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:252)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:242)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:44)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.chann...
[W3C] Matched W3C error code 'no such element' to NoSuchElementError
[BaseDriver] Waited for 266 ms so far
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/m7","context":"","multiple":false}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"ELEMENT":"a0029150-ac75-4aab-ba71-f2763c2504fa","element-6066-11e4-a52e-4f735466cecf":"a0029150-ac75-4aab-ba71-f2763c2504fa"}}
[W3C (1f56446c)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"a0029150-ac75-4aab-ba71-f2763c2504fa","ELEMENT":"a0029150-ac75-4aab-ba71-f2763c2504fa"}
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element 200 1113 ms - 137
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/a0029150-ac75-4aab-ba71-f2763c2504fa/value
[HTTP] {"text":"合肥医保","value":["合","肥","医","保"],"id":"a0029150-ac75-4aab-ba71-f2763c2504fa"}
[W3C (1f56446c)] Calling AppiumDriver.setValue() with args: [["合","肥","医","保"],"a0029150-ac75-4aab-ba71-f2763c2504fa","1f56446c-f400-402a-8ae0-278c1ce22765"]
[WD Proxy] Matched '/element/a0029150-ac75-4aab-ba71-f2763c2504fa/value' to command name 'setValue'
[Protocol Converter] Added 'value' property ["合","肥","医","保"] to 'setValue' request body
[WD Proxy] Proxying [POST /element/a0029150-ac75-4aab-ba71-f2763c2504fa/value] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element/a0029150-ac75-4aab-ba71-f2763c2504fa/value] with body: {"elementId":"a0029150-ac75-4aab-ba71-f2763c2504fa","text":"合肥医保","replace":false,"unicodeKeyboard":true,"value":["合","肥","医","保"]}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":null}
[W3C (1f56446c)] Responding to client with driver.setValue() result: null
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/a0029150-ac75-4aab-ba71-f2763c2504fa/value 200 78 ms - 14
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element
[HTTP] {"using":"id","value":"com.tencent.mm:id/s7"}
[W3C (1f56446c)] Calling AppiumDriver.findElement() with args: ["id","com.tencent.mm:id/s7","1f56446c-f400-402a-8ae0-278c1ce22765"]
[BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[BaseDriver] Waiting up to 10000 ms for condition
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/s7","context":"","multiple":false}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"ELEMENT":"77a0bd55-54f2-4162-96fc-b0516173b062","element-6066-11e4-a52e-4f735466cecf":"77a0bd55-54f2-4162-96fc-b0516173b062"}}
[W3C (1f56446c)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"77a0bd55-54f2-4162-96fc-b0516173b062","ELEMENT":"77a0bd55-54f2-4162-96fc-b0516173b062"}
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element 200 1482 ms - 137
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/77a0bd55-54f2-4162-96fc-b0516173b062/click
[HTTP] {"id":"77a0bd55-54f2-4162-96fc-b0516173b062"}
[W3C (1f56446c)] Calling AppiumDriver.click() with args: ["77a0bd55-54f2-4162-96fc-b0516173b062","1f56446c-f400-402a-8ae0-278c1ce22765"]
[WD Proxy] Matched '/element/77a0bd55-54f2-4162-96fc-b0516173b062/click' to command name 'click'
[WD Proxy] Proxying [POST /element/77a0bd55-54f2-4162-96fc-b0516173b062/click] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element/77a0bd55-54f2-4162-96fc-b0516173b062/click] with body: {"element":"77a0bd55-54f2-4162-96fc-b0516173b062"}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":null}
[W3C (1f56446c)] Responding to client with driver.click() result: null
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/77a0bd55-54f2-4162-96fc-b0516173b062/click 200 60 ms - 14
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element
[HTTP] {"using":"id","value":"com.tencent.mm:id/aqv"}
[W3C (1f56446c)] Calling AppiumDriver.findElement() with args: ["id","com.tencent.mm:id/aqv","1f56446c-f400-402a-8ae0-278c1ce22765"]
[BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[BaseDriver] Waiting up to 10000 ms for condition
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"id","selector":"com.tencent.mm:id/aqv","context":"","multiple":false}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"ELEMENT":"a9f5c0bd-672d-43b3-81b2-8003061102bf","element-6066-11e4-a52e-4f735466cecf":"a9f5c0bd-672d-43b3-81b2-8003061102bf"}}
[W3C (1f56446c)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"a9f5c0bd-672d-43b3-81b2-8003061102bf","ELEMENT":"a9f5c0bd-672d-43b3-81b2-8003061102bf"}
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element 200 1250 ms - 137
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/a9f5c0bd-672d-43b3-81b2-8003061102bf/click
[HTTP] {"id":"a9f5c0bd-672d-43b3-81b2-8003061102bf"}
[W3C (1f56446c)] Calling AppiumDriver.click() with args: ["a9f5c0bd-672d-43b3-81b2-8003061102bf","1f56446c-f400-402a-8ae0-278c1ce22765"]
[WD Proxy] Matched '/element/a9f5c0bd-672d-43b3-81b2-8003061102bf/click' to command name 'click'
[WD Proxy] Proxying [POST /element/a9f5c0bd-672d-43b3-81b2-8003061102bf/click] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element/a9f5c0bd-672d-43b3-81b2-8003061102bf/click] with body: {"element":"a9f5c0bd-672d-43b3-81b2-8003061102bf"}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":null}
[W3C (1f56446c)] Responding to client with driver.click() result: null
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/a9f5c0bd-672d-43b3-81b2-8003061102bf/click 200 52 ms - 14
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element
[HTTP] {"using":"xpath","value":"/hierarchy/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.ListView/android.widget.TextView[2]"}
[W3C (1f56446c)] Calling AppiumDriver.findElement() with args: ["xpath","/hierarchy/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.ListView/android.widget.TextView[2]","1f56446c-f400-402a-8ae0-278c1ce22765"]
[BaseDriver] Valid locator strategies for this request: xpath, id, class name, accessibility id, -android uiautomator
[BaseDriver] Waiting up to 10000 ms for condition
[WD Proxy] Matched '/element' to command name 'findElement'
[WD Proxy] Proxying [POST /element] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element] with body: {"strategy":"xpath","selector":"/hierarchy/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.ListView/android.widget.TextView[2]","context":"","multiple":false}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":{"ELEMENT":"c441ba9c-3064-46ce-8065-24928d7d6151","element-6066-11e4-a52e-4f735466cecf":"c441ba9c-3064-46ce-8065-24928d7d6151"}}
[W3C (1f56446c)] Responding to client with driver.findElement() result: {"element-6066-11e4-a52e-4f735466cecf":"c441ba9c-3064-46ce-8065-24928d7d6151","ELEMENT":"c441ba9c-3064-46ce-8065-24928d7d6151"}
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element 200 690 ms - 137
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/c441ba9c-3064-46ce-8065-24928d7d6151/click
[HTTP] {"id":"c441ba9c-3064-46ce-8065-24928d7d6151"}
[W3C (1f56446c)] Calling AppiumDriver.click() with args: ["c441ba9c-3064-46ce-8065-24928d7d6151","1f56446c-f400-402a-8ae0-278c1ce22765"]
[WD Proxy] Matched '/element/c441ba9c-3064-46ce-8065-24928d7d6151/click' to command name 'click'
[WD Proxy] Proxying [POST /element/c441ba9c-3064-46ce-8065-24928d7d6151/click] to [POST http://localhost:8223/wd/hub/session/e9474135-e485-4f84-872c-8e7392b7a232/element/c441ba9c-3064-46ce-8065-24928d7d6151/click] with body: {"element":"c441ba9c-3064-46ce-8065-24928d7d6151"}
[WD Proxy] Got response with status 200: {"sessionId":"e9474135-e485-4f84-872c-8e7392b7a232","value":null}
[W3C (1f56446c)] Responding to client with driver.click() result: null
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/element/c441ba9c-3064-46ce-8065-24928d7d6151/click 200 681 ms - 14
[HTTP] 
[HTTP] --> GET /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/contexts
[HTTP] {}
[W3C (1f56446c)] Calling AppiumDriver.getContexts() with args: ["1f56446c-f400-402a-8ae0-278c1ce22765"]
[AndroidDriver] Getting a list of available webviews
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell cat /proc/net/unix'
[AndroidDriver] Not checking whether webviews have active pages; use the 'ensureWebviewsHavePages' cap to turn this check on
[AndroidDriver] WEBVIEW_31932 mapped to pid 31932
[AndroidDriver] Getting process name for webview
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell ps'
[AndroidDriver] Parsed pid: '31932' pkg: 'com.tencent.mm:tools' from
[AndroidDriver]     USER           PID  PPID     VSZ    RSS WCHAN            ADDR S NAME
[AndroidDriver]     u0_a401      31932   639 2623924 265644 0                   0 S com.tencent.mm:tools
[AndroidDriver] Returning process name: 'com.tencent.mm:tools'
[AndroidDriver] Found webviews: ["WEBVIEW_com.tencent.mm:tools"]
[AndroidDriver] Available contexts: ["NATIVE_APP","WEBVIEW_com.tencent.mm:tools"]
[W3C (1f56446c)] Responding to client with driver.getContexts() result: ["NATIVE_APP","WEBVIEW_com.tencent.mm:tools"]
[HTTP] <-- GET /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/contexts 200 195 ms - 55
[HTTP] 
[HTTP] --> POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/context
[HTTP] {"name":"WEBVIEW_com.tencent.mm:tools"}
[W3C (1f56446c)] Calling AppiumDriver.setContext() with args: ["WEBVIEW_com.tencent.mm:tools","1f56446c-f400-402a-8ae0-278c1ce22765"]
[AndroidDriver] Getting a list of available webviews
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell cat /proc/net/unix'
[AndroidDriver] Not checking whether webviews have active pages; use the 'ensureWebviewsHavePages' cap to turn this check on
[AndroidDriver] WEBVIEW_31932 mapped to pid 31932
[AndroidDriver] Getting process name for webview
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell ps'
[AndroidDriver] Parsed pid: '31932' pkg: 'com.tencent.mm:tools' from
[AndroidDriver]     USER           PID  PPID     VSZ    RSS WCHAN            ADDR S NAME
[AndroidDriver]     u0_a401      31932   639 2619848 267272 0                   0 S com.tencent.mm:tools
[AndroidDriver] Returning process name: 'com.tencent.mm:tools'
[AndroidDriver] Found webviews: ["WEBVIEW_com.tencent.mm:tools"]
[AndroidDriver] Available contexts: ["NATIVE_APP","WEBVIEW_com.tencent.mm:tools"]
[AndroidDriver] Connecting to chrome-backed webview context 'WEBVIEW_com.tencent.mm:tools'
[AndroidDriver] A port was not given, using random free port: 8000
[AndroidDriver] Automated Chromedriver download is disabled. Use 'chromedriver_autodownload' server feature to enable it
[AndroidDriver] Before starting chromedriver, androidPackage is 'com.tencent.mm'
[Chromedriver] Changed state to 'starting'
[Chromedriver] Found 1 executable in 'C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-chromedriver\chromedriver\win'
[Chromedriver] The following Chromedriver executables were found:
[Chromedriver]     'C:/Users/蒋磊/AppData/Local/Programs/Appium/resources/app/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe' (version '80.0.3987.16', minimum Chrome version 'Unknown')
[ADB] Getting package info for 'com.android.chrome'
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 shell dumpsys package com.android.chrome'
[Chromedriver] Found Chrome bundle 'com.android.chrome' version '80.0.3987'
[Chromedriver] No known Chromedriver available to automate Chrome version '80.0.3987'.
[Chromedriver] Using Chromedriver version '80.0.3987.16', which has not been tested with Appium. You could also try to enable automated chromedrivers download server feature
[Chromedriver] Set chromedriver binary as: C:/Users/蒋磊/AppData/Local/Programs/Appium/resources/app/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe
[Chromedriver] Killing any old chromedrivers, running: wmic process where "commandline like '%chromedriver.exe%--port=8000%'" delete
[Chromedriver] Successfully cleaned up old chromedrivers
[Chromedriver] Cleaning any old adb forwarded port socket connections
[ADB] List forwarding ports
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 forward --list'
[ADB] Removing forwarded port socket connection: 11862 
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 forward --remove tcp\:11862'
[ADB] Removing forwarded port socket connection: 11874 
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 forward --remove tcp\:11874'
[ADB] Removing forwarded port socket connection: 11891 
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 forward --remove tcp\:11891'
[ADB] Removing forwarded port socket connection: 11903 
[ADB] Running 'D:\APP\Android-sdk\platform-tools\adb.exe -P 5037 -s S2D0218B09012250 forward --remove tcp\:11903'
[Chromedriver] Spawning chromedriver with: C:/Users/蒋磊/AppData/Local/Programs/Appium/resources/app/node_modules/appium/node_modules/appium-chromedriver/chromedriver/win/chromedriver.exe --url-base=wd/hub --port=8000 --adb-port=5037 --verbose
[Chromedriver] Chromedriver version: '80.0.3987.16'
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8000/wd/hub/status] with no body
[WD Proxy] Got response with status 200: {"value":{"build":{"version":"80.0.3987.16 (320f6526c1632ad4f205ebce69b99a062ed78647-refs/branch-heads/3987@{#185})"},"message":"ChromeDriver ready for new sessions.","os":{"arch":"x86_64","name":"Windows NT","version":"10.0.17763"},"ready":true}}
[Chromedriver] Starting W3C Chromedriver session with capabilities: {
[Chromedriver]   "capabilities": {
[Chromedriver]     "alwaysMatch": {
[Chromedriver]       "goog:chromeOptions": {
[Chromedriver]         "androidPackage": "com.tencent.mm",
[Chromedriver]         "androidUseRunningApp": true,
[Chromedriver]         "androidProcess": "com.tencent.mm:tools",
[Chromedriver]         "androidDeviceSerial": "S2D0218B09012250"
[Chromedriver]       },
[Chromedriver]       "goog:loggingPrefs": {
[Chromedriver]         "browser": "ALL"
[Chromedriver]       }
[Chromedriver]     }
[Chromedriver]   }
[Chromedriver] }
[WD Proxy] Matched '/session' to command name 'createSession'
[WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"capabilities":{"alwaysMatch":{"goog:chromeOptions":{"androidPackage":"com.tencent.mm","androidUseRunningApp":true,"androidProcess":"com.tencent.mm:tools","androidDeviceSerial":"S2D0218B09012250"},"goog:loggingPrefs":{"browser":"ALL"}}}}
[Chromedriver] Webview version: 'Chrome/66.0.3359.126'
[WD Proxy] Got an unexpected response with status 500: {"value":{"error":"session not created","message":"session not created: This version of ChromeDriver only supports Chrome version 80","stacktrace":"Backtrace:\n\tOrdinal0 [0x003F0AE3+1706723]\n\tOrdinal0 [0x003568C1+1075393]\n\tOrdinal0 [0x002CDFA9+516009]\n\tOrdinal0 [0x00260554+66900]\n\tOrdinal0 [0x0025DBE7+56295]\n\tOrdinal0 [0x0025B7E5+47077]\n\tOrdinal0 [0x0027BFD7+180183]\n\tOrdinal0 [0x0027BDDD+179677]\n\tOrdinal0 [0x00279D4B+171339]\n\tOrdinal0 [0x00261D4A+73034]\n\tOrdinal0 [0x00262DC0+77248]\n\tOrdinal0 [0x00262D59+77145]\n\tOrdinal0 [0x0036BB37+1162039]\n\tGetHandleVerifier [0x0048A7C6+508998]\n\tGetHandleVerifier [0x0048A504+508292]\n\tGetHandleVerifier [0x0049F617+594583]\n\tGetHandleVerifier [0x0048B036+511158]\n\tOrdinal0 [0x00363FFC+1130492]\n\tOrdinal0 [0x0036D49B+1168539]\n\tOrdinal0 [0x0036D603+1168899]\n\tOrdinal0 [0x00385B05+1268485]\n\tBaseThreadInitThunk [0x75950419+25]\n\tRtlGetAppContainerNamedObjectPath [0x76F3662D+237]\n\tRtlGetAppContainerNamedObjectPath [0x76F365FD+189]\n"}}
[W3C] Matched W3C error code 'session not created' to SessionNotCreatedError
[Chromedriver] Failed to start Chromedriver session: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80
[WD Proxy] Matched '/session' to command name 'createSession'
[WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"capabilities":{"alwaysMatch":{"goog:chromeOptions":{"androidPackage":"com.tencent.mm","androidUseRunningApp":true,"androidProcess":"com.tencent.mm:tools","androidDeviceSerial":"S2D0218B09012250"},"goog:loggingPrefs":{"browser":"ALL"}}}}
[Chromedriver] Webview version: 'Chrome/66.0.3359.126'
[WD Proxy] Got an unexpected response with status 500: {"value":{"error":"session not created","message":"session not created: This version of ChromeDriver only supports Chrome version 80","stacktrace":"Backtrace:\n\tOrdinal0 [0x003F0AE3+1706723]\n\tOrdinal0 [0x003568C1+1075393]\n\tOrdinal0 [0x002CDFA9+516009]\n\tOrdinal0 [0x00260554+66900]\n\tOrdinal0 [0x0025DBE7+56295]\n\tOrdinal0 [0x0025B7E5+47077]\n\tOrdinal0 [0x0027BFD7+180183]\n\tOrdinal0 [0x0027BDDD+179677]\n\tOrdinal0 [0x00279D4B+171339]\n\tOrdinal0 [0x00261D4A+73034]\n\tOrdinal0 [0x00262DC0+77248]\n\tOrdinal0 [0x00262D59+77145]\n\tOrdinal0 [0x0036BB37+1162039]\n\tGetHandleVerifier [0x0048A7C6+508998]\n\tGetHandleVerifier [0x0048A504+508292]\n\tGetHandleVerifier [0x0049F617+594583]\n\tGetHandleVerifier [0x0048B036+511158]\n\tOrdinal0 [0x00363FFC+1130492]\n\tOrdinal0 [0x0036D49B+1168539]\n\tOrdinal0 [0x0036D603+1168899]\n\tOrdinal0 [0x00385B05+1268485]\n\tBaseThreadInitThunk [0x75950419+25]\n\tRtlGetAppContainerNamedObjectPath [0x76F3662D+237]\n\tRtlGetAppContainerNamedObjectPath [0x76F365FD+189]\n"}}
[W3C] Matched W3C error code 'session not created' to SessionNotCreatedError
[Chromedriver] Failed to start Chromedriver session: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80
[WD Proxy] Matched '/session' to command name 'createSession'
[WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"capabilities":{"alwaysMatch":{"goog:chromeOptions":{"androidPackage":"com.tencent.mm","androidUseRunningApp":true,"androidProcess":"com.tencent.mm:tools","androidDeviceSerial":"S2D0218B09012250"},"goog:loggingPrefs":{"browser":"ALL"}}}}
[Chromedriver] Webview version: 'Chrome/66.0.3359.126'
[WD Proxy] Got an unexpected response with status 500: {"value":{"error":"session not created","message":"session not created: This version of ChromeDriver only supports Chrome version 80","stacktrace":"Backtrace:\n\tOrdinal0 [0x003F0AE3+1706723]\n\tOrdinal0 [0x003568C1+1075393]\n\tOrdinal0 [0x002CDFA9+516009]\n\tOrdinal0 [0x00260554+66900]\n\tOrdinal0 [0x0025DBE7+56295]\n\tOrdinal0 [0x0025B7E5+47077]\n\tOrdinal0 [0x0027BFD7+180183]\n\tOrdinal0 [0x0027BDDD+179677]\n\tOrdinal0 [0x00279D4B+171339]\n\tOrdinal0 [0x00261D4A+73034]\n\tOrdinal0 [0x00262DC0+77248]\n\tOrdinal0 [0x00262D59+77145]\n\tOrdinal0 [0x0036BB37+1162039]\n\tGetHandleVerifier [0x0048A7C6+508998]\n\tGetHandleVerifier [0x0048A504+508292]\n\tGetHandleVerifier [0x0049F617+594583]\n\tGetHandleVerifier [0x0048B036+511158]\n\tOrdinal0 [0x00363FFC+1130492]\n\tOrdinal0 [0x0036D49B+1168539]\n\tOrdinal0 [0x0036D603+1168899]\n\tOrdinal0 [0x00385B05+1268485]\n\tBaseThreadInitThunk [0x75950419+25]\n\tRtlGetAppContainerNamedObjectPath [0x76F3662D+237]\n\tRtlGetAppContainerNamedObjectPath [0x76F365FD+189]\n"}}
[W3C] Matched W3C error code 'session not created' to SessionNotCreatedError
[Chromedriver] Failed to start Chromedriver session: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80
[WD Proxy] Matched '/session' to command name 'createSession'
[WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8000/wd/hub/session] with body: {"capabilities":{"alwaysMatch":{"goog:chromeOptions":{"androidPackage":"com.tencent.mm","androidUseRunningApp":true,"androidProcess":"com.tencent.mm:tools","androidDeviceSerial":"S2D0218B09012250"},"goog:loggingPrefs":{"browser":"ALL"}}}}
[Chromedriver] Webview version: 'Chrome/66.0.3359.126'
[WD Proxy] Got an unexpected response with status 500: {"value":{"error":"session not created","message":"session not created: This version of ChromeDriver only supports Chrome version 80","stacktrace":"Backtrace:\n\tOrdinal0 [0x003F0AE3+1706723]\n\tOrdinal0 [0x003568C1+1075393]\n\tOrdinal0 [0x002CDFA9+516009]\n\tOrdinal0 [0x00260554+66900]\n\tOrdinal0 [0x0025DBE7+56295]\n\tOrdinal0 [0x0025B7E5+47077]\n\tOrdinal0 [0x0027BFD7+180183]\n\tOrdinal0 [0x0027BDDD+179677]\n\tOrdinal0 [0x00279D4B+171339]\n\tOrdinal0 [0x00261D4A+73034]\n\tOrdinal0 [0x00262DC0+77248]\n\tOrdinal0 [0x00262D59+77145]\n\tOrdinal0 [0x0036BB37+1162039]\n\tGetHandleVerifier [0x0048A7C6+508998]\n\tGetHandleVerifier [0x0048A504+508292]\n\tGetHandleVerifier [0x0049F617+594583]\n\tGetHandleVerifier [0x0048B036+511158]\n\tOrdinal0 [0x00363FFC+1130492]\n\tOrdinal0 [0x0036D49B+1168539]\n\tOrdinal0 [0x0036D603+1168899]\n\tOrdinal0 [0x00385B05+1268485]\n\tBaseThreadInitThunk [0x75950419+25]\n\tRtlGetAppContainerNamedObjectPath [0x76F3662D+237]\n\tRtlGetAppContainerNamedObjectPath [0x76F365FD+189]\n"}}
[W3C] Matched W3C error code 'session not created' to SessionNotCreatedError
[Chromedriver] Failed to start Chromedriver session: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80
[Chromedriver] Chromedriver exited unexpectedly with code null, signal SIGTERM
[Chromedriver] Changed state to 'stopped'
[Chromedriver] Error: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80
[Chromedriver]     at Object.wrappedLogger.errorAndThrow (C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-support\lib\logging.js:79:13)
[Chromedriver]     at Chromedriver.errorAndThrow [as start] (C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:540:11)
[W3C (1f56446c)] Encountered internal error running command: Error: A new session could not be created. Details: session not created: This version of ChromeDriver only supports Chrome version 80
[W3C (1f56446c)]     at Object.wrappedLogger.errorAndThrow (C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-support\lib\logging.js:79:13)
[W3C (1f56446c)]     at Chromedriver.errorAndThrow [as start] (C:\Users\蒋磊\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-chromedriver\lib\chromedriver.js:540:11)
[HTTP] <-- POST /wd/hub/session/1f56446c-f400-402a-8ae0-278c1ce22765/context 500 4155 ms - 869
[HTTP]

**Comment by mykola-mokhnach** [comment]
Duplicate of https://github.com/appium/appium/issues/13215
